src/lib/k8s/webhook.ts

```diff
@@ -453,7 +453,7 @@ export class Webhook {
         name: this.name,
         description: `Pepr Module: ${this.config.description}`,
         url: "https://github.com/defenseunicorns/pepr",
-        version: `${this.config.version}`,
+        version: `${this.config.appVersion || "0.0.1"}`,
```

Comment:
defaulting to 0.0.1 seems dangerous, don't we want it to be "unknown" or something, does it have to be \d+\.\d+\.\d+

Reply:
This is just the user app version, it's only used as an optional metadata field for the zarf package. By default it starts at 0.0.1 unless the break something.